src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5475,6 +5475,83 @@
                 "path": "https://creativecommons.org/licenses/by/4.0/"
             }],
             "sources": []
+        },
+        {
+            "profile": "tabular-data-resource",
+            "name": "assn_eia_epacems",
+            "path": "data/plants_eia.csv",
```

Comment:
You'll need to update the path here to refer to the correct output CSV file which it looks like will be `data/assn_eia_epacems.csv`